### ISSUE TITLE
Cannot install ember-cli@0.2.3 on Windows

### ISSUE THREAD

**Issue by joelalejandro** [ember-cli/ember-cli#5510]
When I install `ember-cli` version 0.2.3 with `npm install -g ember-cli@0.2.3`, the process fails with the following errors:

```
npm WARN EBUNDLEOVERRIDE Replacing bundled node_modules\ember-cli\node_modules\broccoli with new installed version
npm WARN EBUNDLEOVERRIDE Replacing bundled node_modules\ember-cli\node_modules\express with new installed version
npm WARN EBUNDLEOVERRIDE Replacing bundled node_modules\ember-cli\node_modules\morgan with new installed version
npm verb stack Error: ENOENT: no such file or directory, chmod 'D:\TFS\Turnero Universal\EmberApp\node_modules\ember-cli\node_modules\broccoli\node_modules\handlebars\bin\handlebars'
npm verb stack     at Error (native)
npm verb cwd D:\TFS\Turnero Universal\EmberApp
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\nodejs\\node.exe" "C:\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--verbose"
npm ERR! node v5.5.0
npm ERR! npm  v3.3.12
npm ERR! path D:\TFS\Turnero Universal\EmberApp\node_modules\ember-cli\node_modules\broccoli\node_modules\handlebars\bin\handlebars
npm ERR! code ENOENT
npm ERR! errno -4058
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod 'D:\TFS\Turnero Universal\EmberApp\node_modules\ember-cli\node_modules\broccoli\node_modules\handlebars\bin\handlebars'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent
npm verb exit [ -4058, true ]
```

It's a consistent behaviour for a freshly cloned project. I've done `npm cache clean` before installing as well, all to no avail. 

I cannot use another version due to dependency constraints.

Log is here: [npm-debug.log.txt](https://github.com/ember-cli/ember-cli/files/143232/npm-debug.log.txt)

**Comment by rwjblue** [comment]
There are many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many many bugs that have been fixed since 0.2.3.  I doubt there is much we can do at this point to help support you other than to suggest that you upgrade...

**Comment by joelalejandro** [comment]
I've been talking to the guys over Slack regarding this issue, I had misinterpreted some stuff regarding ember-cli and ember versions. Nevermind this issue.

Lol, yes, @rwjblue, I'm aware of the maaaaaaaaaaaaaaany things that are fixed. I normally work with the 2.x branch by now, this is an old project for which I can't allocate too much time.
